pkg/redis/Tests/RedisConnectionFactoryConfigTest.php

```diff
@@ -264,11 +264,11 @@ public static function provideConfigs()
             ],
         ];
 
-        //check tls connection for predis library
+        // check tls connection for predis library
         yield [
             'rediss+phpredis://localhost:1234?foo=bar&async=1',
             [
-                'host' => 'tls://localhost',
+                'host' => 'localhost',
```

Comment:
could you elaborate on this change ?

Reply:
@makasim I was just trying to make the tests to pass :heavy_check_mark: 
The `master` branch is also failing on these test after your merged https://github.com/php-enqueue/enqueue-dev/pull/1297 and I don't see the tests run on this PR before being merged :thinking: 
![image](https://github.com/php-enqueue/enqueue-dev/assets/37912048/d49dae20-b76e-4e23-85bc-f6b9b336b356)

Reply:
that's strange indeed